core/src/main/java/io/questdb/cairo/map/OrderedMap.java

```diff
@@ -916,6 +927,15 @@ public void putLong256(Long256 value) {
             appendAddress += 32L;
         }
 
+        @Override
+        public void putLong256(long l0, long l1, long l2, long l3) {
+            Unsafe.getUnsafe().putLong(appendAddress, l0);
```

Comment:
We need `checkCapacity(32L);` here - otherwise we may segfault to due insufficient heap size.